lua/telescope/_extensions/aerial.lua

```diff
@@ -59,12 +60,16 @@ local function aerial_picker(opts)
   end
 
   local function make_entry(item)
-    local name = item.name
-    if show_nesting then
-      local cur = item.parent
-      while cur do
-        name = string.format("%s.%s", cur.name, name)
-        cur = cur.parent
+    if opts.get_entry_text ~= nil then
+      name = opts.get_entry_text(item)
+    else
+      local name = item.name
```

Comment:
You need to add a `local name` before the `if...else` and remove the `local` here

Reply:
done now... initialized it to item.name.. in theory I could initialize to nil or not initialize at all.. not sure what's best.

Reply:
There's basically no difference.